### PR TITLE
LRCI-3714 When a test fails in the setUp block of a Poshi test, import into testray with BLOCKED status 

### PR DESCRIPTION
**Pull request by CalumR23**
https://liferay.atlassian.net/browse/LRCI-3714
https://liferay.atlassian.net/browse/POSHI-705

### CODE REVIEW

**Comment by CalumR23** [comment]
ci:test:sf

File: modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/PoshiRunner.java [hunk 1]
```diff
@@ -205,10 +205,17 @@ public void setUp() throws Exception {
...
 			throw webDriverException;
 		}
-		catch (Exception exception) {
+		catch (Exception exception1) {
 			LiferaySeleniumUtil.printJavaProcessStacktrace();
 
-			_throwException(exception);
+			StringBuilder sb = new StringBuilder();
+
+			sb.append("Failed setup with: ");
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/PoshiRunner.java:213]: Can we change the message to: 
`TEST_SETUP_ERROR: `

So it's more unique. We have something like this already: 
https://github.com/liferay/liferay-portal/blob/master/modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/LiferaySeleniumUtil.java#L111

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by kenjiheigel** [comment]
ci:forward

**Pull request closed without merging**